share/man/man5/rc.conf.5

```diff
@@ -618,6 +618,15 @@ When specified, this variable overrides the
 variable for interface
 .Ar iface
 only.
+.It Va dhclient_arpwait
+.Pq Vt bool
+Set to
+.Dq Li YES
```

Comment:
YES is the default, so for `rc.conf(5)` we should say what happens if we set it to NO.